local_test_env.yml

```diff
@@ -0,0 +1,13 @@
+# This file is used to install packages for local testing
+name: chairlift-test
+channels:
+  - snowflake
+dependencies:
+  - python=3.8
+  - snowflake-snowpark-python
+  - pytest
+  - pip
+  - pip:
+      - pytest
```

Comment:
intentionally repeated?

Reply:
if you mean pip, yes. When I remove it, I get a warning recommending to explicitly add pip.

Reply:
No, I will remove it.